qa/files.txt

```diff
@@ -1 +0,0 @@
-github_refresh.py
```

Comment:
I guess you can add src directory here for test coverage

Reply:
thats in the directory.txt. Have added it there